CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- :boom: `AsyncTypeAheadInput` based on MUI Autocomplete component.
```

Comment:
not sure how easy and straight forward, but would it be a lot of work for a short "how to migrate" for this two components?

Reply:
thank you :pray: